classes/Http/Controller/TalkController.php

```diff
@@ -380,7 +406,17 @@ public function updateAction(Request $req)
 
         if ($isValid) {
             $sanitized_data = $form->getCleanData();
-            $updated_at = new \DateTime();
+            $tags = explode(',', $sanitized_data['tags']);
+
+            $tag_entity = [];
```

Comment:
Actually already working on refactoring a it a bit to convert it from an inline code to a method.  I don't know if an exception really needs to be thrown if a user does not provide tags for there talks.  It's not a required field at this point in time.